app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -2,7 +2,7 @@
 class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
   queue_as :wakes_facebook_metrics
 
-  def perform(locations)
+  def perform(*locations)
```

Comment:
@gaganawhad if not required, I think we remove, right? Can you see some reason to leave it in?